applications/RomApplication/python_scripts/element_selection_strategy.py

```diff
@@ -0,0 +1,30 @@
+
+class ElementSelectionStrategy():
+    def __init__(self):
+        """ The required tolerances are entered here. Default values to be provided
```

Comment:
this base class intends to abstrac the element selection steps, so that we can add other element selection algorithms. Each element selection algorithm shuould provide their own parameters